mlflow/store/db_migrations/versions/a1b2c3d4e5f6_add_spans_table.py

```diff
@@ -56,23 +94,54 @@ def upgrade():
         sa.PrimaryKeyConstraint("trace_id", "span_id", name="spans_pk"),
     )
 
+    if bind.dialect.name == "postgresql" and not use_generated_column:
+        from sqlalchemy import text
+
+        op.execute(
+            text(
+                """
+            CREATE OR REPLACE FUNCTION update_span_duration()
```

Comment:
Are there any permissions issues we need to be wary of for creating / replacing triggers & functions?